deepchem/utils/typing.py

```diff
@@ -7,8 +7,8 @@
 # An activation function for a Keras layer: either a TensorFlow function or the name of a standard activation
 KerasActivationFn = Union[Callable, str]
```

Comment:
Should we rename the other parameters to remove `Keras` (eg. `KerasActivationFn` -> `ActivationFn`?

Reply:
This one is specific to Keras.  It refers to how layers let you write either `activation=tf.nn.relu` or `activation='relu'`.